### ISSUE TITLE
Building features with class but no subtype

### ISSUE THREAD

**Issue by jeffdefacto** [OvertureMaps/data#183]
In the latest release candidate, buildings with class: bridge_structure have no subtype associated with them.

<img width="423" alt="image" src="https://github.com/user-attachments/assets/01491f20-a0a2-43ee-b02d-44bcb23f7fd1">

**Comment by atiannicelli** [comment]
What subtype should we use?

**Comment by LukaszMichalski-TomTom** [comment]
@jeffdefacto yes that is my mistake. I added bridge_structure class but I didn't check if there is correct type for it.

**Comment by jenningsanderson** [comment]
Should we add a "structure" `subtype` or do we add this to "transportation" ? Or something else? 

(7/23 buildings tf meeting)

**Comment by atiannicelli** [comment]
I have added a subtype of "transportation" for anything with a class of bridge_structure that didn't have a subtype previously.
[see the diff here](https://github.com/OvertureMaps/schema/pull/258/files/3f1644b585f2737eb5c2a4f542c7ab2bec2e0c2c#diff-4c9dc7d021b37570a261c105e47394ceeaa92c21ba2161438590b6ebeb24e530R222)

this should be fixed in August release
